src/arrayMethodSort.js

```diff
@@ -3,9 +3,39 @@
 /**
  * Implement method Sort
  */
+const compareStrings = (a, b) => {
+  const strA = String(a);
+  const strB = String(b);
+
+  if (strA > strB) {
+    return 1;
+  } else if (strA === strB) {
+    return 0;
+  } else {
+    return -1;
+  }
```

Comment:
You can return a boolean value, cause it converts to a number in the `> 0` comparison.
```suggestion
  return strA > strB;
```